recipe/disable_autolink_windows.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/include/GL/freeglut_std.h b/include/GL/freeglut_std.h
```

Comment:
What's the reason for the patch? A header would be nice

Reply:
The reason for the patch is explained [here](https://github.com/conda-forge/freeglut-feedstock/issues/26#issuecomment-769383551) but is relevant if CMake relies on `-G "%CMAKE_GENERATOR%"`. As I replaced it with `ninja` we can drop the patch.

Reply:
Re-enabled the patch to fix the usage of `freeglut.lib`